### PR TITLE
Changed debounce to higher interval to fix the input bounce problem

### PR DESCRIPTION
**Pull request by alex-solo**
### Description

Tiny change to fix the issue of the cursor bouncing left and right after every input change.

### What to test for/How to test

In the currency field, the cursor should no longer move left or right and maintain its position at the last entered or deleted value.

### CODE REVIEW

File: components/EligibilityPage/index.tsx [hunk 1]
```diff
@@ -261,7 +261,7 @@ export const EligibilityPage: React.VFC = ({}) => {
                 label={field.config.label}
                 onChange={debounce(
                   (e) => handleOnChange(field, e.target.value),
-                  100
+                  500
```
Comment (MarcoGoC) [inline components/EligibilityPage/index.tsx]: although it helps, still has issues when entering the income very slow, not sure why is this happening as older branches didn't have this issue

**Review by MarcoGoC — CHANGES_REQUESTED** [review]
not sure why is this happening

File: components/EligibilityPage/index.tsx (hunk 1 above)
Comment (alex-solo) [reply components/EligibilityPage/index.tsx]: hmm, I tested again and it seems to be working fine on initial input but when going back to the input, the bouncing back and forth comes back. Darn.. I'm not sure what caused this to start happening but I will investigate a bit more.

Comment (alex-solo) [reply components/EligibilityPage/index.tsx]: Ah, I simply needed to move the setState for error visibility after React actually deals with changing the input state!

**Review by MarcoGoC — APPROVED** [review]
works as before +1

**Pull request merged**